app/models/pickup.rb

```diff
@@ -0,0 +1,4 @@
+class Pickup < ActiveRecord::Base
```

Comment:
I think it's just leads to easier modeling... The other way around we'll probably add review prefix to each attribute